inductiva/simulation/simulator.py

```diff
@@ -39,29 +57,25 @@ def run(
             **kwargs: Additional keyword arguments to be passed to the
                 simulation API method.
         """
-        input_dir = files.resolve_path(input_dir)
-        if not input_dir.is_dir():
-            raise ValueError(
-                f"The provided path (\"{input_dir}\") is not a directory.")
-
-        if output_dir is None:
-            output_dir = input_dir.with_name(f"{input_dir.name}-output")
-            output_dir = files.get_timestamped_path(output_dir)
+        input_dir, output_dir = self._setup_dirs(input_dir, output_dir)
+        if pipeline is None:
+            params = [kwargs]
```

Comment:
Why wrapping `kwargs` with `[]`?

Reply:
Because our idea was to pass the params to the api as a list of arguments, where each member of the list is a dictionary with parameters for the specific command. Then the API would iterate over the different commands. But this isn't yet implemented in the backend.